Resources/config/services.xml

```diff
@@ -40,6 +40,7 @@
             <argument type="service" id="sulu_redirect.import"/>
             <argument type="string">%sulu_redirect.imports.path%</argument>
 
+            <tag name="controller.service_arguments"/>
```

Comment:
think we are not using `service_arguments`, so would just set the service to `public`

Reply:
Is there a reason for not using `controller.service_arguments`?

Reply:
i think we just decided to use `public` when switching to controllers as services. maybe is a bit faster too, have not investigated that.

Reply:
but you added this because you need the service to be public, right? so i think its a bit confusing to use the `controller.service_arguments` tag when we do not intend to use the feature behind it

Reply:
okay will change that 👍